includes/paragraphs.inc

```diff
@@ -114,17 +106,17 @@ function oe_bootstrap_theme_preprocess_paragraph__oe_text_feature_media(array &$
   // Get the media source.
   $source = $media->getSource();
 
-  // If it's an image media, render it and assign it to the image variable.
-  if ($source instanceof MediaAvPortalPhotoSource || $source instanceof Image) {
-    $thumbnail = $media->get('thumbnail')->first();
-    $variables['image'] = ImageValueObject::fromStyledImageItem($thumbnail, 'oe_bootstrap_theme_medium_no_crop');
-    $cacheability->applyTo($variables);
-    // Overwrite the variant because is an image.
-    $variables['variant'] = 'image';
+  $is_image = $source instanceof MediaAvPortalPhotoSource || $source instanceof Image;
+  $is_video = $source instanceof MediaAvPortalVideoSource || $source instanceof OEmbed || $source instanceof Iframe;
+
+  // If it's no image or no video, bail out.
```

Comment:
```suggestion
  // If it's not an image or not a video, bail out.
```

Reply:
In fact, must be: // If it's not an image **and** not a video, bail out.